src/pages/HomePage.tsx

```diff
@@ -46,10 +47,11 @@ const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
   useEffect(() => {
     if (match.params.lng) {
       store.changeLanguage(match.params.lng);
+      setLanChanged(!lanChanged);
```

Comment:
setLanChanged(prevLanChanged => !prevLanChanged);

Reply:
I still need to see where the problem happened.
My solution need to refix 😉 

Thanks !